core/src/main/java/com/exadel/etoolbox/backpack/core/services/pckg/impl/BasePackageServiceImpl.java

```diff
@@ -142,15 +147,21 @@ public boolean isEnableStackTrace() {
      */
     @Override
     public PackageInfo getPackageInfo(final ResourceResolver resourceResolver, final PackageModel packageModel) {
-        List<String> actualPaths = packageModel.getPaths().stream()
-                .filter(s -> resourceResolver.getResource(s.getPath()) != null)
-                .map(path -> getActualPath(path.getPath(), path.isExcludeChildren(), resourceResolver))
-                .collect(Collectors.toList());
+        List<String> actualPaths;
+        if (packageModel.getPaths() != null) {
```

Comment:
Better use `CollectioinUtils.isNotEmpty(packageModel.getPaths())`. I'm not quite sure they'll be always null, but may be an empty collection

Reply:
ok, I'll include this library in future projects